visivo/models/selector.py

```diff
@@ -0,0 +1,47 @@
+from enum import Enum
+from pydantic import ConfigDict, Field, PrivateAttr, model_serializer
+
+from visivo.models.base.base_model import BaseModel
+from visivo.models.base.named_model import NamedModel
+
+
+class SelectorType(str, Enum):
+    single = "single"
+    multiple = "multiple"
+
+
+class Selector(NamedModel, BaseModel):
+    """
+    Selectors can be used to add interactivity between charts and tables.
+
+    Below is how you would link two charts to show the same selected data.
```

Comment:
Yay for docs! We could add a a gif of your six second video in the docs too.

Reply:
Does the video live in the docs here or on "interactivity"?